steam/steam.go

```diff
@@ -57,3 +66,44 @@ func GetDiscounts(db *sql.DB) ([]SteamGame, error) {
 	}
 	return res, nil
 }
+
+// Get all featured games in current featured table.
+func GetFeatured(db *sql.DB, feature string) ([]SteamGame, error) {
+	if !IsValidFeature(feature) {
+		feature = "win"
+	}
+	queryOneFeature := fmt.Sprintf(
+		"%s where feature_type='featured_%s'", queryAllFeatured, feature)
+	rows, err := db.Query(queryOneFeature)
+
+	if err != nil {
+		return nil, err
+	}
+
+	defer rows.Close()
+
+	res := make([]SteamGame, 0)
+	for rows.Next() {
+		var game SteamGame
+
+		err = rows.Scan(
+			&game.Name, &game.URL, &game.ImgSrc, &game.Review, &game.PriceBefore,
```

Comment:
since both discount / featured games are using `game.Review` but with different meanings (one for review one for headline), consider changing it to something like `SteamGame.Text`?